src/pages/en/reference/api-reference.md

```diff
@@ -166,6 +166,31 @@ Astro.response.headers.set('Set-Cookie', 'a=b; Path=/;');
 ---
 ```
 
+### `Astro.cookies`
+
```

Comment:
Ah, we should probably include this to flag the version this API was added in. Could help anyone on an older version who's looking at these docs.

```suggestion

<Since v="1.4.0" />

```